lib/scss_lint/reporters/codeclimate_reporter.rb

```diff
@@ -24,7 +24,9 @@ def report_lints
             }
           }
         }.to_json
-      end.join("\0")
+        $stdout << "#{issue_json}\0"
```

Comment:
Do we use `#<<` for printing elsewhere? Do we explicitly call methods on `$stdout`? Both seem like patterns we don't typically do

```rb
print "#{issue_json}\0"
```

I'd could also get behind:

```rb
print {
  # ...
}.to_json
print "\0"
```